sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -718,6 +719,8 @@ def process(self, element, load_job_name_prefix, *schema_side_inputs):
         source_format=self.source_format,
         job_labels=self.bq_io_metadata.add_additional_bq_job_labels(),
         load_job_project_id=self.load_job_project_id)
+    yield pvalue.TaggedOutput(
+        TriggerLoadJobs.ONGOING_JOBS, (destination, job_reference))
     self.pending_jobs.append(
```

Comment:
When does this get invoked (since we yield above) ?

Reply:
I believe yield doesn't return/exit out of process, so we would still continue to the next line.

Reply:
Ack. LG.